internal/commands/app/create_inputs.go

```diff
@@ -18,10 +18,9 @@ var (
 	flagDirectoryShort = "c"
 	flagDirectoryUsage = "the directory to create your new Realm app, defaults to Realm app name"
 
-	// TODO(REALMC-8135): Implement data-source flag for app create command
-	// flagDataSource      = "data-source"
-	// flagDataSourceShort = "s"
-	// flagDataSourceUsage = "atlas cluster to back your Realm app, defaults to first available"
+	flagDataSource      = "data-source"
+	flagDataSourceShort = "s"
+	flagDataSourceUsage = "atlas cluster to back your Realm app, defaults to first available"
```

Comment:
`include to link an Atlas cluster to your Realm app, defaults to first available`